.travis.yml

```diff
@@ -4,8 +4,10 @@ node_js:
 cache:
   directories:
   - node_modules
+env:
+  - SOLIDITY_COVERAGE=true
```

Comment:
You need to have two envs so that you get one job with the normal tests and another one with the additional migration instrumentation.

https://github.com/OpenZeppelin/zeppelin-solidity/blob/a7e91856f3e275668b4a4c55cbd14864aa61b100/.travis.yml#L10-L12

Additionally you can make the build finish when the tests have finished running, but coverage is still running, with these lines:

https://github.com/OpenZeppelin/zeppelin-solidity/blob/a7e91856f3e275668b4a4c55cbd14864aa61b100/.travis.yml#L13-L16

This is nice because coverage usually takes a lot more time to run.

Reply:
beautiful, fixing now.